### ISSUE TITLE
Editor crash when opening PluginScript resource

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#46566]
**Godot version:**
Godot 3.2.4 rc3

**Issue description:**
```
editor/editor_node.cpp:1950:58: runtime error: member call on null pointer of type 'struct ScriptLanguage'
editor/editor_node.cpp:1950:58: runtime error: member access within null pointer of type 'struct ScriptLanguage'
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] godots() [0x1736270] (/mnt/Miecz/godot3.2/platform/x11/crash_handler_x11.cpp:54)
[2] /lib/x86_64-linux-gnu/libc.so.6(+0x46210) [0x7fda76f43210] (??:0)
[3] godots() [0x88c6706] (/mnt/Miecz/godot3.2/editor/editor_node.cpp:1950)
[4] EditorNode::_edit_current() (/mnt/Miecz/godot3.2/editor/editor_node.cpp:2096 (discriminator 24))
[5] EditorNode::push_item(Object*, String const&, bool) (/mnt/Miecz/godot3.2/editor/editor_node.cpp:1911)
[6] InspectorDock::_resource_selected(Ref<Resource> const&, String const&) const (/mnt/Miecz/godot3.2/editor/inspector_dock.cpp:285)
[7] InspectorDock::edit_resource(Ref<Resource> const&) (/mnt/Miecz/godot3.2/editor/inspector_dock.cpp:370 (discriminator 2))
[8] EditorNode::load_resource(String const&, bool) (/mnt/Miecz/godot3.2/editor/editor_node.cpp:949)
[9] FileSystemDock::_select_file(String const&, bool) (/mnt/Miecz/godot3.2/editor/filesystem_dock.cpp:840)
[10] FileSystemDock::_tree_activate_file() (/mnt/Miecz/godot3.2/editor/filesystem_dock.cpp:854 (discriminator 12))
[11] MethodBind0::call(Object*, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/./core/method_bind.gen.inc:61 (discriminator 4))
[12] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/core/object.cpp:919 (discriminator 1))
[13] Object::emit_signal(StringName const&, Variant const**, int) (/mnt/Miecz/godot3.2/core/object.cpp:1246 (discriminator 1))
[14] Object::emit_signal(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/mnt/Miecz/godot3.2/core/object.cpp:1303)
[15] Tree::_gui_input(Ref<InputEvent>) (/mnt/Miecz/godot3.2/scene/gui/tree.cpp:2736 (discriminator 1))
[16] MethodBind1<Ref<InputEvent> >::call(Object*, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/./core/method_bind.gen.inc:775 (discriminator 12))
[17] Object::call_multilevel(StringName const&, Variant const**, int) (/mnt/Miecz/godot3.2/core/object.cpp:761 (discriminator 1))
[18] Object::call_multilevel(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/mnt/Miecz/godot3.2/core/object.cpp:862)
[19] Viewport::_gui_call_input(Control*, Ref<InputEvent> const&) (/mnt/Miecz/godot3.2/scene/main/viewport.cpp:1716 (discriminator 1))
[20] Viewport::_gui_input_event(Ref<InputEvent>) (/mnt/Miecz/godot3.2/scene/main/viewport.cpp:2033 (discriminator 3))
[21] Viewport::input(Ref<InputEvent> const&) (/mnt/Miecz/godot3.2/scene/main/viewport.cpp:2923 (discriminator 2))
[22] Viewport::_vp_input(Ref<InputEvent> const&) (/mnt/Miecz/godot3.2/scene/main/viewport.cpp:1480)
[23] MethodBind1<Ref<InputEvent> const&>::call(Object*, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/./core/method_bind.gen.inc:775 (discriminator 12))
[24] Object::call(StringName const&, Variant const**, int, Variant::CallError&) (/mnt/Miecz/godot3.2/core/object.cpp:919 (discriminator 1))
[25] Object::call(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/mnt/Miecz/godot3.2/core/object.cpp:846)
[26] SceneTree::call_group_flags(unsigned int, StringName const&, StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/mnt/Miecz/godot3.2/scene/main/scene_tree.cpp:276)
[27] SceneTree::input_event(Ref<InputEvent> const&) (/mnt/Miecz/godot3.2/scene/main/scene_tree.cpp:432 (discriminator 5))
[28] InputDefault::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/mnt/Miecz/godot3.2/main/input_default.cpp:429)
[29] InputDefault::parse_input_event(Ref<InputEvent> const&) (/mnt/Miecz/godot3.2/main/input_default.cpp:273)
[30] InputDefault::flush_accumulated_events() (/mnt/Miecz/godot3.2/main/input_default.cpp:692)
[31] OS_X11::process_xevents() (/mnt/Miecz/godot3.2/platform/x11/os_x11.cpp:2324)
[32] OS_X11::run() (/mnt/Miecz/godot3.2/platform/x11/os_x11.cpp:3626)
[33] godots(main+0x326) [0x172cefc] (/mnt/Miecz/godot3.2/platform/x11/godot_x11.cpp:57)
[34] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fda76f240b3] (??:0)
[35] godots(_start+0x2e) [0x172cb1e] (??:?)
```
**Steps to reproduce:**
1. Open file from file system dock - `PluginScript.res` - it was saved with ResourceSaver

**Minimal reproduction project:**
[PluginScript.res.zip](https://github.com/godotengine/godot/files/6062796/PluginScript.res.zip)

**Comment by KoBeWi** [comment]
Might be related to #31769

**Comment by akien-mga** [comment]
Fixed by #66298.
